map/bookmark_manager.cpp

```diff
@@ -2220,10 +2238,11 @@ void BookmarkManager::UpdateBmGroupIdList()
     return lhs.second->GetLastModifiedTime() > rhs.second->GetLastModifiedTime();
   });
 
-  m_bmGroupsIdList.clear();
-  m_bmGroupsIdList.resize(count);
-  for (size_t i = 0; i < count; ++i)
-    m_bmGroupsIdList[i] = vec[i].first;
+  for (size_t i = 0; i < count; ++i) {
+    sortedList.push_back(vec[i].first);
+  }
```

Comment:
One-liners without bracers. If with bracers, they're on a new line in C++ (it was the same in ObjC until Swift arrived).
```suggestion
  for (size_t i = 0; i < count; ++i)
    sortedList.push_back(vec[i].first);
```